_data/clients/profanity.yml

```diff
@@ -1,6 +1,16 @@
 name: Profanity
 url: http://profanity.im/
 tracking_issue: https://github.com/boothj5/profanity/issues/658
+features:
+  single_chat: true
+  group_chat: true
+  encrypted_files: true
+os:
+  mac: true
+  win: true
+  linux: true
+  android: true
+  ios: true
```

Comment:
Really?

Reply:
No. That was just used to showcase the possibilities and profanity was chosen randomly.

Reply:
I think @mimi89999 implies that this could be more of a 
```
os:
  - mac
  - win
  - linux
```
kinda-list. If it's missing from that list, it'll be false. 😄

Reply:
@ReneVolution remember, omemo.top is not a complete XMPP clients details sheet, it's the tracker for OMEMO-related things.